examples/vl3_basic/Makefile

```diff
@@ -27,3 +27,11 @@ docker-vl3: build-vl3
 	cp -p ${GOPATH}/bin/linux_amd64/vl3_nse ${DOCKER_BUILD_TOP}/vl3/
 	cp -pr ./examples/universal-cnf/vppagent/etc/* ${DOCKER_BUILD_TOP}/vl3/etc/
 	cd ${DOCKER_BUILD_TOP}/vl3 && docker build -t ${ORG}/vl3_ucnf-vl3-nse:${TAG} --build-arg VPP_AGENT=${VPP_AGENT} -f Dockerfile.no_go_build_ucnf .
+
+.PHONY: docker-vl3-ipam
```

Comment:
Remove this target.  You can just build with a different TAG to make a new version of the docker-vl3 NSE.  We don't want separate image types.